### PR TITLE
Refactor/css/media/ow

### LINKED ISSUE #7: Meet the specs listed below under “MVP”
- [x] Assume one single user - multiple users with log in and authentication is out of the scope of this project.

- [x] The application cannot simply be a display of data - there needs to be some way for the user to work with or manipulate the data (favoriting, searching, filtering, commenting, rating, etc)

- [x] The application should be a multi-page application using React Router

- [x] Be able to answer: What problems are you solving? What features must the solution include to solve this problem?

### PR DESCRIPTION
**Pull request by oakleywindiate**
### Description
* Summary: 
1) Delete sample files in Cypress
2) Correct cypress testing for dropdown menu
3) Correct CSS styling for meowdy by moving App-title into `background-events`

* Where should the reviewer start:
`App.tsx`

* Background context:
* Questions: 

### Relevant Issues

Resolves #7 

### Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

#

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -44,6 +35,30 @@
...
-.routes {
-  display: flex;
+.speech-bubble,
+.meowdy {
+  display: none;
+  position: fixed;
+  color: #b76027;
 }
+
+@media screen and (max-width: 75em) {
```
Comment (emmacbd) [inline src/App.css:45]: Thanks for adding this!

File: src/App.css [hunk 2]
```diff
@@ -4,36 +4,27 @@
...
   margin: 0;
   position: fixed;
   top: 0;
+  display: inline-block;
   left: 50%;
   font-size: 7em;
   padding-top: .2em;
 }
 
-.meowdy
```
Comment (angelewilliams) [inline src/App.css:16]: Glad we got meowdy back!

**Review by shanekwarning — APPROVED** [review]
Great work with the keys and the meowdy. Thank you for adding dynamic styling as well.

File: src/CatContainer.tsx [hunk 3]
```diff
@@ -12,24 +12,22 @@ type CatData = {
 
 const CatContainer: React.FC<CatData> = ({ catData, searchCats }) => {
 
-  const catTiles = catData.map(cat => {
+  const catTiles = catData.map((cat) => {
     return (
-      <NavLink className="cat-tiles-nav" to={`/cats/${cat.id}`}>
+      <NavLink className="cat-tiles-nav" key={cat.id} to={`/cats/${cat.id}`}>
```
Comment (angelewilliams) [inline src/CatContainer.tsx:17]: Glad we resolved the error about the key!

**Review by emmacbd — APPROVED** [review]
Looks great!

**Review by angelewilliams — APPROVED** [review]

**Pull request merged**